index.html

```diff
@@ -2,11 +2,522 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <!-- <link rel="icon" href="%PUBLIC_URL%/favicon.ico" /> -->
```

Comment:
Remove commented code

Reply:
Hello. for the footer section we did as the original one. Try to see in desktop size(min-width: 1224px). For screen size (max-width: 1224px) we changes to other type of footer.